cmd/minikube/cmd/delete.go

```diff
@@ -723,3 +730,36 @@ func getPids(path string) ([]int, error) {
 
 	return pids, nil
 }
+
+func deleteKnownHosts() error {
+	cname := ClusterFlagValue()
+	co, err := mustload.RunningWithError(cname)
+	if err != nil {
+		return err
+	}
+	if co.CP.Host.DriverName == driver.None {
+		// none' driver does not support 'minikube ssh-host' command
+		// so nothing needs to be done
+		return nil
+	}
+	var n *config.Node
+	if nodeName == "" {
+		n = co.CP.Node
+	} else {
+		n, _, err = node.Retrieve(*co.Config, nodeName)
+		if err != nil {
+			return fmt.Errorf("node %s does not exist: %v", nodeName, err)
+		}
+	}
+	// acquire the current key line
+	keyLine, err := machine.RunSSHHostCommand(co.API, *co.Config, *n, "ssh-keyscan", []string{"-t", "rsa"})
```

Comment:
let try to see if we can get this key locally in the minikube home folder

for example maybe this one ?
 ~/.minikube/machines/minikube/id_rsa.pub